tools/sigh.js

```diff
@@ -240,7 +240,7 @@ function test(args) {
     // TODO(wkorman): Integrate shell testing more deeply into sigh testing. For
     // now we skip including shell tests in the normal sigh test flow and intend
     // to instead run them via a separate 'npm test' command.
-    if (fullPath.startsWith(fixPathForWindows(`${dir}/shell/`))) return false;
+    if (fullPath.startsWith(path.normalize(`${dir}/shell/`))) return false;
```

Comment:
LOL, obviously `fixPathForWindows` won't really fix the path for Windows.

Reply:
fixPathForWindows takes environment paths and "fixes" them to become standardized paths (i.e. the opposite of normalize?). It only operates on Windows paths, because Linux paths are already standardized.